google/externalaccount/filecredsource.go

```diff
@@ -26,7 +26,7 @@ func (cs fileCredentialSource) credentialSourceType() string {
 func (cs fileCredentialSource) subjectToken() (string, error) {
 	tokenFile, err := os.Open(cs.File)
 	if err != nil {
-		return "", fmt.Errorf("oauth2/google/externalaccount: failed to open credential file %q", cs.File)
+		return "", fmt.Errorf("oauth2/google/externalaccount: failed to open credential file %q due to %v", cs.File, err)
```

Comment:
a common pattern is to use `<text>: %v` instead. Suggested change:

```go
return "", fmt.Errorf("oauth2/google/externalaccount: failed to open credential file %q: %v", cs.File, err)
```

Reply:
```suggestion
		return "", fmt.Errorf("oauth2/google/externalaccount: failed to open credential file %q: %v", cs.File, err)
```